### DISCUSSION TITLE
Different fluences in 1st catalog and Repeaters pages

### DISCUSSION THREAD

**Discussion by fverrecchia** [chime-frb-open-data/community#26] [Q&A] [answered]
We note that the fluences reported in the first CHIME/FRB catalogue dedicated
web page on your site

https://www.chime-frb.ca/catalog

are significatly different from those reported in the "Repeating FRBs" section of the site

https://www.chime-frb.ca/repeaters

of course for those bursts for which fluences are available in this last section.

## Expected Behavior

We suppose the same fluence  values for the same burst should appear un both pages

## Current Behavior

For instance here are four cases:

- FRB20180814A, first burst:
Fluence in catalog: 3.0±1.0 [Jy ms]
Fluence in "repeaters": 21(15) [Jy ms]

- FRB20180916B:

° first burst:
Fluence in catalog: 6.0±2.0
Fluence in "repeaters": 2.3 (1.2) [Jy ms]

° second burst (181019):
Fluence in catalog: 10.0±3.0 [Jy ms]
Fluence in "repeaters": 2.7 (1.3) [Jy ms]

- FRB20181017A, first burst:
Fluence in catalog: 16.0±6.0 [Jy ms]
Fluence in "repeaters": 1 (0.5) [Jy ms]

so in some case they are marginally compatible but in various cases at all. 

## Context

We used the fluences in the Repeaters page for our work and now we found out that in the 1st catalog they seem wrong.

 Thank you very much.
Francesco

**Comment by shinybrar** [comment]
@fverrecchia Thank you for reporting this issue, we have started to investigate this internally and will most likely provide an update as soon as possible.

**Comment by fverrecchia** [comment]
Dear Shiny and all, do you have any news? Thanks a lot. 
Francesco

**Comment by shinybrar** [comment, marked as answer]
@fverrecchia Apologies, I thought this was already answered by a colleague of mine, but here is our current understanding,

# Status
- Internally there are very strong indications after doing some re-checking that the fluences published in the Catalog Paper are correct and supersede any previously published data.
- The uncertainties of some catalog fluences don't overlap some previously published results and believe that due to the improved data analysis techniques (e.g. DM optimization) and better understanding of the primary shape of the beam compared to 2018.
- Additionally, our algorithms are significantly better now at excising radio frequency interference from our data, which is evident in the plots of FRB20180814A

  **Before**
  ![](https://storage.googleapis.com/chimefrb-dev.appspot.com/FRB20180814A/180917_waterfall.png)

  **After**
  ![](https://storage.googleapis.com/chimefrb-dev.appspot.com/FRB20180917A/waterfall_hires.png)

## Moving Forward
First of all, we would really like to thank you @fverrecchia and your collaborators in bringing this issue to our attention. Moving forward we plan to take the following actions in the coming weeks,
1. Update https://www.chime-frb.ca/repeaters to depict the updated fluence from the catalog, rather than the original repeater paper.
2. Publish an updated catalogue v1.1 down the road, with detailed remarks about changes (issues other have reported) and the peculiarities of the dataset (issues you have reported).
3. Note that you will always be able download any previous catalog release and the associated changelog for it so that you can track the changes of the dataset within your individual research.

**Comment by fverrecchia** [comment]
Dear Shiny, thank you very much! We are happy to have helped. At any rate, regarding the entries in https://www.chime-frb.ca/repeaters only, do you plan some update of the results?
Cheers, Francesco

**Reply by fverrecchia** [reply]
Dear Shiny and All, I repeat the question: do you plan to update also the results only present at  https://www.chime-frb.ca/repeaters ?
Moreover for many bursts fluence is not published in that page, are there any possibility to get these information?
Thank you very much.
Cheers, Francesco

**Reply by emilypetroff** [reply]
Thanks @fverrecchia for checking in on this. Someone from our team will cross-check these events in the next few weeks. @Astronomer-Mohit can you please have a look?

**Reply by fverrecchia** [reply]
Dear Emily and All, do you have any news? Thank you. Best, Francesco

**Reply by emilypetroff** [reply]
Hi Francesco, thanks for the check in. Can I just confirm with you how urgently you need this and for what purpose? From my understanding of the request, you'd like all the flux and fluence values for repeater bursts published as part of Catalog 1, and that are currently listed on the individual FRB pages at https://www.chime-frb.ca/catalog to also be populated into the tables for each source on the repeaters page https://www.chime-frb.ca/repeaters . Is that correct?

This is quite a big task on our side. We are making plans to do it, but it will take some time. If this is urgently needed and it's not sufficient to go to extract this information from the catalog 1 csv (note that there is a parameter "repeater_name" that allows you to sort bursts from different repeaters), please provide more detail here, and we can try to accelerate this internally. Thanks.

**Comment by fverrecchia** [comment]
Hi Emily,

we are interested in having the largest number of burst fluences is possible for a sample of R-FRBs, giving priority to those from the Catalog 1 as you told us. Those bursts which are also in "repeaters" web page have better parameters in Cat1.
So the questions for you are:
- the fluences that are in "repeatears" web page ONLY are reliable or are you going to update them?
- do you plan to add further fluences to the "repeaters" web page for those bursts which are present only there?

we are continuing our X&gamma-ray counterpart search project with AGILE data as you know, so we were working to a further paper, as we wrote also to Vicky some time ago.
We should review our work if no radio fluences are available, but surely we can understand your problem.
Probably the last point is the most important.
Thank you very much.
Best,

Francesco
